templates/master/00-master/two-node-with-fencing/units/ha-00-directories.service.yaml

```diff
@@ -0,0 +1,17 @@
+name: ha-00-directories.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Create Directories and Log Files for PCS and Corosync
+  After=network.target
+  
```

Comment:
I've noticed that we also have trailing whitespace embedded in this files. Might be worth cleaning that up.